README.MD

```diff
@@ -55,23 +61,75 @@ Only basic features that almost all projects use, were added in this template:
 
 ## Project Setup
 
-### CI
+### Signing Builds
 
-This project uses Github Actions + fastlane for CI. There is a nice upgrade path to automatically
-deploy builds to Firebase App Distribution and Play Store.
+If you need local signing of release builds, copy the upload keystore into the root of your project,
+and call it 'upload-keystore.jks'. Note that you probably don't need to use local signing. The CI
+build setup builds and signs your app for
+you.
 
-#### Installing fastlane
+### CI / Automated Builds
 
-A big advantage of fastlane is that the ci and publishing steps can run locally.
-Use `brew install fastlane` and then `fastlane test` to verify this installation.
+This project uses Github Actions for CI. We have added these workflows for now:
 
-#### Removing Github Actions + fastlane
+- _debug.yml_: any commit on any branch triggers a `devDebug` apk and a `prodDebug` apk build
+- _release.yml_: any PR triggers signed release builds (`prodRelease` bundle and `prodRelease` apk)
 
-Using Bitrise instead? Then you can delete:
+#### Adding your own keystore's details on Github Actions
+
+- First create your own keystore and store it in 1pw. _You do not add this keystore to the repo_ (
+  like we did in this template as an example).
+- add a KEYSTORE_BASE_64 [new github repository secret](./settings/secrets/actions/new) that is the
```

Comment:
After merging this PR, please trigger ask dependabot build by choosing any open dependabot PR and reacting with `@dependabot recreate` 

If that fails, the changes you made in the secrets need to be reflected into the dependabot secrets; and that should be documented here

Reply:
See https://github.com/Q42/Template.Android/settings/secrets/dependabot